platform/core/PolyPodCoreSwift/Tests/PolyPodCoreSwiftTests/CodableTests/MessagePackEncoderTests.swift

```diff
@@ -0,0 +1,228 @@
+//
+//  MessagePackEncoderTests.swift
+//  
+//
+//  Created by Palade Timotei on 14.08.22.
+//
+
+import XCTest
+import MessagePack
+
+@testable import PolyPodCoreSwift
+
+class MessagePackEncoderTests: XCTestCase {
```

Comment:
It will be nice to have some tests for encoding custom types, like structs and enums.